Woodoku/src/ui/command_line_ui.py

```diff
@@ -1,29 +1,88 @@
 from typing import List, Tuple
 
+from elements import *
 from entity.woodoku_board import WoodokuBoard
 from entity.woodoku_shape import WoodokuShape
 from ui.ui_interface import UIInterface
 
+NUM_SHAPES = 3
+
 
 class CommandLineUI(UIInterface):
 
     def show_start_game(self, board: WoodokuBoard) -> None:
-        pass
+        print(orange("\nWelcome to Wooduku"))
+        print(board)
 
     def show_board(self, board: WoodokuBoard) -> None:
-        pass
+        print(board)
 
     def choose_shape(self, shapes: List[WoodokuShape], shape_availability: List[bool]) -> int:
-        pass
+        shapes_str = ""
+        for idx, avai in enumerate(shape_availability):
+            if avai:
+                shapes_str += f"{idx}.\n{str(shapes[idx]).rstrip()}\n\n"
+        print(shapes_str)
+        val = ""
+        while not isinstance(val, int) or val not in range(NUM_SHAPES):
+            val = input(f"Please choose one of the {orange('shapes')}:\n")
+            try:
+                val = int(val)
```

Comment:
Though Python is a dynamic typed language, I am still against the idea of changing variable's type. Thoughts? @Xiaorui-Huang

Reply:
Agree. from line 27 to 35 needs to be refactored and I think it's unnecessary to raise an Error here

Reply:
If we don't raise an error here, what is the behavior of not getting a value from 0 to 2?

Reply:
Yeah the error is needed. User could input a string instead of integer as well.